src/openrct2/ride/Vehicle.cpp

```diff
@@ -5788,500 +5788,10 @@ GForces Vehicle::GetGForces() const
-            break;
-        case TrackElemType::Up60ToUp25:     //
-        case TrackElemType::Down25ToDown60: //
-        case TrackElemType::Up60ToUp25Covered:
-        case TrackElemType::Down25ToDown60Covered:
-            vertFactor = -82;
-            // 6d7551
-            break;
-        case TrackElemType::LeftQuarterTurn5Tiles: //
-        case TrackElemType::LeftQuarterTurn5TilesUp25:
-        case TrackElemType::LeftQuarterTurn5TilesDown25:
-        case TrackElemType::LeftTwistDownToUp:
-        case TrackElemType::LeftTwistUpToDown:
-        case TrackElemType::LeftQuarterTurn5TilesCovered:
-        case TrackElemType::LeftQuarterHelixLargeUp:
-        case TrackElemType::LeftQuarterHelixLargeDown:
-        case TrackElemType::LeftFlyerTwistUp:
-        case TrackElemType::LeftFlyerTwistDown:
-        case TrackElemType::LeftHeartLineRoll:
-            lateralFactor = 98;
-            // 6d7590
-            break;
-        case TrackElemType::RightQuarterTurn5Tiles: //
-        case TrackElemType::RightQuarterTurn5TilesUp25:
-        case TrackElemType::RightQuarterTurn5TilesDown25:
-        case TrackElemType::RightTwistDownToUp:
-        case TrackElemType::RightTwistUpToDown:
-        case TrackElemType::RightQuarterTurn5TilesCovered:
-        case TrackElemType::RightQuarterHelixLargeUp:
-        case TrackElemType::RightQuarterHelixLargeDown:
-        case TrackElemType::RightFlyerTwistUp:
-        case TrackElemType::RightFlyerTwistDown:
-        case TrackElemType::RightHeartLineRoll:
-            lateralFactor = -98;
-            // 6d75B7
-            break;
-        case TrackElemType::BankedLeftQuarterTurn5Tiles:
-        case TrackElemType::LeftHalfBankedHelixUpLarge:
-        case TrackElemType::LeftHalfBankedHelixDownLarge:
-        case TrackElemType::LeftQuarterBankedHelixLargeUp:
-        case TrackElemType::LeftQuarterBankedHelixLargeDown:
-            vertFactor = 200;
-            lateralFactor = 160;
-            // 6d75E1
-            break;
-        case TrackElemType::BankedRightQuarterTurn5Tiles:
-        case TrackElemType::RightHalfBankedHelixUpLarge:
-        case TrackElemType::RightHalfBankedHelixDownLarge:
-        case TrackElemType::RightQuarterBankedHelixLargeUp:
-        case TrackElemType::RightQuarterBankedHelixLargeDown:
-            vertFactor = 200;
-            lateralFactor = -160;
-            // 6d75F0
-            break;
-        case TrackElemType::SBendLeft:
-        case TrackElemType::SBendLeftCovered:
-            lateralFactor = (track_progress < 48) ? 98 : -98;
-            // 6d75FF
-            break;
-        case TrackElemType::SBendRight:
-        case TrackElemType::SBendRightCovered:
-            lateralFactor = (track_progress < 48) ? -98 : 98;
-            // 6d7608
-            break;
-        case TrackElemType::LeftVerticalLoop:
-        case TrackElemType::RightVerticalLoop:
-            vertFactor = (abs(track_progress - 155) / 2) + 28;
-            // 6d7690
```

Comment:
@duncanspumpkin with this PR these address comments are removed. Is that alright?

Reply:
They will have no use so sure thats fine.